folium/map.py

```diff
@@ -5,6 +5,9 @@
 
 Classes for drawing maps.
 """
+
+from __future__ import unicode_literals
```

Comment:
So we can use `"Ça c&#39;est chouette"` instead of `u"Ça c&#39;est chouette"`.  Embrace the future :wink: